### PR TITLE
[INFRA-251] Allow two Syncano lib version in codebox

### PR DESCRIPTION
**Pull request by kotchaosu**
Deprecated version run with:

```
docker run -it quay.io/syncano/python-codebox test.py
```

or

```
docker run -it quay.io/syncano/python-codebox python test.py
```

The new version

```
docker run -it quay.io/syncano/python-codebox python27-42 test.py
```

### CODE REVIEW

**Comment by 23doors** [comment]
Looks good but I would suggest a little different naming:
New version should probably be the default one so:
`docker run -it quay.io/syncano/python-codebox python test.py`
and separate old version instead with a versioned directory (in case we ever need more versions):
so e.g.:
`docker run -it quay.io/syncano/python-codebox /home/syncano/v4.1/bin/python test.py`

Also, it may be more future-proof to maintain requirements for both libraries separately rather than expect that library version is the only difference.

Changing default python will need to be handled gracefully:
- first make it so both `python` and `/home/syncano/v4.1/bin/python` use 4.1.0 library (don't install 4.1.0). \* change command in our backend to use v4.1.0
- deploy to production. 
- then change change the image to support 4.2.0 by default and add new runtime to backend.

Otherwise - if we change the default version we will need to have some downtime to deploy both this and new platform at once. To prevent this in future, maybe we should also version 4.2.0 in the same way? At least in the way our platform runs it. For convenience we could define aliases:

```
alias python27-41=/home/syncano/v4.1/bin/python
alias python27-42=/home/syncano/v4.2/bin/python
alias python=python27-42
```

so we got `docker run -it quay.io/syncano/python-codebox python27-41 test.py`

File: Dockerfile [hunk 1]
```diff
@@ -15,17 +14,33 @@ RUN apt-get update && apt-get install -qqy \
...
-    wget \
-    && wget https://bootstrap.pypa.io/get-pip.py \
-    && python get-pip.py \
-    && pip install --upgrade pip \
-    && pip install -r /tmp/requirements.txt \
-    && pip install -r /tmp/external_requirements.txt
+    wget && \
+    wget https://bootstrap.pypa.io/get-pip.py && \
+    python get-pip.py && \
+    pip install --upgrade pip && \
```
Comment (23doors) [inline Dockerfile]: Isn't get-pip installing latest version anyway?

**Comment by 23doors** [comment]
looks good!

File: Dockerfile (hunk 1 above)
Comment (kotchaosu) [inline Dockerfile]: it used to be different, i'll check again

Comment (kotchaosu) [inline Dockerfile]: it's not necessary

**Pull request merged**